ext/SCPyPlotExt.jl

```diff
@@ -48,17 +50,17 @@ function pgdsgui(ax::PyCall.PyObject, ridata::AbstractVector{Pair{Union{Method,M
     return methodref
 end
 
-function pgdsgui(ax::PyCall.PyObject, args...; consts::Bool=true, by=inclusive, kwargs...)
+function SnoopCompile.pgdsgui(ax::PyCall.PyObject, args...; consts::Bool=true, by=inclusive, kwargs...)
     pgdsgui(ax, prep_ri(args...; consts, by, kwargs...); bystr=get_bystr(by), consts, kwargs...)
 end
 
-function pgdsgui(args...; kwargs...)
+function SnoopCompile.pgdsgui(args...; kwargs...)
     fig, ax = plt.subplots()
     pgdsgui(ax, args...; kwargs...), ax
 end
 
 function prep_ri(tinf::InferenceTimingNode, pdata=Profile.fetch(); lidict=lookups, consts, by, kwargs...)
-    lookup_firstip!(lookups, pdata)
+    SnoopCompile.lookup_firstip!(lookups, pdata)
     return runtime_inferencetime(tinf, pdata; lidict, consts, by)
```

Comment:
```suggestion
    SnoopCompile.lookup_firstip!(lookups, pdata)
    return SnoopCompile.runtime_inferencetime(tinf, pdata; lidict, consts, by)
```
I don't see an `export` for `runtime_inferencetime` either